pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -82,6 +83,11 @@ func (ku *KubeUtil) init() error {
 		}
 	}
 
+	ku.podResources, err = NewPodResourcesClient()
+	if err != nil {
+		log.Debugf("Failed to create pod resources client, resource data will not be available: %s", err)
```

Comment:
Depending on usage, we should probably make it a `warning` or display something in status.

Reply:
Changed to a warning. How could we add it to the status?